src/python/pants/engine/native.py

```diff
@@ -376,6 +376,12 @@ def extern_store_i64(self, context_handle, i64):
     c = self._ffi.from_handle(context_handle)
     return c.to_value(i64)
 
+  @_extern_decl('Handle', ['ExternContext*', 'double'])
+  def extern_store_f64(self, context_handle, f64):
```

Comment:
extern_store_double? (not a blocking comment though)

Reply:
The "double" bit is only how C would describe this... and it's not particularly descriptive. It looks like`C`/`C++` don't have type declarations equivalent to `uint64_t` for floating point... which is a bummer: https://stackoverflow.com/questions/2524737/fixed-size-floating-point-types

But I think the method name is right, and more descriptive.